### PR TITLE
Fixed bug in name search and pagination

### PR DESCRIPTION
**Pull request by melanahammel**

### CODE REVIEW

File: cerberus-ip-translator-lambda/src/main/java/com.nike.cerberus.lambda.waf/CerberusMetadataLookup.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ private OkHttpClient createHttpClient() {
 
         try {
             Request request = new Request.Builder()
-                .url(cerberusUrl + "/v1/metadata?limit=2000&offset=0")
+                .url(cerberusUrl + "/v1/metadata?limit=10000&offset=0")
```
Comment (fieldju) [inline cerberus-ip-translator-lambda/src/main/java/com.nike.cerberus.lambda.waf/CerberusMetadataLookup.java]: What is this, this call should be a do while loop until there is no next page?

Comment (fieldju) [reply cerberus-ip-translator-lambda/src/main/java/com.nike.cerberus.lambda.waf/CerberusMetadataLookup.java]:
```
def res = list
def hasNext = false
do {
 def cur = fetch the results
 res += cur.stuff you care about
 hasNext = cur.hasNext
} while there is no next page
```

Comment (mayitbeegh) [reply cerberus-ip-translator-lambda/src/main/java/com.nike.cerberus.lambda.waf/CerberusMetadataLookup.java]: I think 10000 is fine until there's a problem. 
But then I just realized this is on public repo. 😅

Comment (melanahammel) [reply cerberus-ip-translator-lambda/src/main/java/com.nike.cerberus.lambda.waf/CerberusMetadataLookup.java]: Will implement this

**Pull request merged**